site/public/js/forum.js

```diff
@@ -2417,6 +2417,37 @@ function clearReplyBoxAutosave(replyBox) {
     }
 }
 
+function setupDisableReplyThreadForm() {
+    const threadPostForms = document.querySelectorAll('.thread-post-form');
+
+    threadPostForms.forEach((form) => {
+        //For all thread forms either reply's or posts, ensure that when text area is empty, the submit button appears to be disabled
+        const textArea = form.querySelector('textarea');
+        const submitButton = form.querySelector('input[type="submit"]');
+
+        if (textArea.id === 'reply_box_0' || textArea.id === 'reply_box_1') {
```

Comment:
#10109 is going to shift these from 0 and 1 to 1 and 2. If that gets merged first be sure to update this. Otherwise I'll adjust that when this gets merged.